src/main/java/com/app/onlinebookstore/dto/user/UserLoginRequestDto.java

```diff
@@ -0,0 +1,13 @@
+package com.app.onlinebookstore.dto.user;
+
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotBlank;
+import org.hibernate.validator.constraints.Length;
+
+public record UserLoginRequestDto(@NotBlank
+                                  @Email
+                                  String email,
+                                  @NotBlank
+                                  @Length(min = 8, max = 20)
```

Comment:
read about the difference between `@Size` and `@Length` and choose the best option.